smrf/distribute/image_data.py

```diff
@@ -129,7 +129,7 @@ def _initialize(self, topo, metadata):
         try:
             self.mx = metadata.utm_x.values
             self.my = metadata.utm_y.values
-        except:
+        except Exception:
```

Comment:
Couldn't this below call also raise an exception?

Reply:
This is some old stuff for the database that I'd like to remove from SMRF #127. This would simply a lot and we don't use the database to run anymore.